### PR TITLE
convert jupyter notebooks to sphynx tutorial

### PR DESCRIPTION
**Pull request by abhiskk**
## Motivation and Context

Converts jupyter notebooks to sphynx tutorials. This should be done because github doesn't work well with jupyter notebooks

## Types of changes

- Docs change / refactoring / dependency upgrade

## TODO
- [x] insert links for notebook downloads.
- [x] update habitat website once PR is ready.
- [x] move habitat-sim notebook also under here: https://github.com/facebookresearch/habitat-sim/blob/master/notebooks/habitat-sim-demo.ipynb
- [x] merge upstream master.
- [x] open jupyter notebook removal PR in sim.

### CODE REVIEW

**Comment by mathfac** [comment]
@abhiskk thank you for taking care of this. I really like the idea to move notebooks out of the main repo. With current approach looks like with changes we have no mechanism to test that notebooks and update them and documentation accordingly.
Where do you plan to save that notebooks?
I propose to create separate repo for Habitat examples and tutorials that showcase everything we have built. Then we can have CI testing of that repo for each PR.

**Comment by abhiskk** [comment]
@mathfac creating another repository for example is definitely a possibility, although right now as our examples and documentation is not extensive I would suggest going forward with the current setup.

File: README.md [hunk 1]
```diff
@@ -1,14 +1,13 @@
 [![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/facebookresearch/habitat-api/blob/master/LICENSE)
 [![CircleCI](https://circleci.com/gh/facebookresearch/habitat-api.svg?style=shield)](https://circleci.com/gh/facebookresearch/habitat-api)
-[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/facebookresearch/habitat-api/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)
```
Comment (mathfac) [inline README.md]: Why we decided remove this shield? Looks like some people are using `good+first+issue` label as staring point.

File: README.md [hunk 2]
```diff
@@ -119,7 +118,7 @@ An important objective of Habitat-API is to make it easy for users to set up a v
 * `Episode`: a class for episode specification that includes the initial position and orientation of an Agent, a scene id, a goal position and optionally shortest paths to the goal. An episode is a description of one task instance for the agent.
 
 <p align="center">
-  <img src='res/img/habitat-api_structure.png' alt="teaser results" width="100%"/>
```
Comment (mathfac) [inline README.md]: Looks like `habitat-api_structure.png` wasn't removed.

**Comment by abhiskk** [comment]
@mathfac  can you explain in more details what do you mean by "Copy pasting cell by cell making them non usable." I am getting rid of the jupyter notebook completely and have just hosted if someone else wants to download it.

In the future we should move to colab notebook but uploading jupyter notebooks to github is definitely a bad decision.

File: README.md (hunk 1 above)
Comment (abhiskk) [reply README.md]: A github user understands how PRs work and issues is a good place to start making contribution. We don't need to explicitly state this and this is redundant information in the README.

**Pull request merged**